pkg/nodeconfig/nodeconfig.go

```diff
@@ -263,14 +263,15 @@ func (nc *nodeConfig) SafeReboot(ctx context.Context) error {
 	if err := nc.Windows.RebootAndReinitialize(); err != nil {
 		return err
 	}
-	// wait for WICD to remove Reboot annotation
```

Comment:
The changes are distinct, this commit adjusts WMCO's behavior and the next addresses WICD. I can squash though since the changes are closely tied together